### PR TITLE
[JENKINS-41517] More consistent logging

### PR DESCRIPTION
**Pull request by stephenc**
See [JENKINS-41517](https://issues.jenkins-ci.org/browse/JENKINS-41517)

@reviewbybees

### CODE REVIEW

**Comment by rsandell** [comment]
:bee:

**Comment by rsandell** [comment]
Didn't look into the "nullability" of all the values being logged, but they seem to be never null to be at least.

File: src/main/java/jenkins/branch/MultiBranchProject.java [hunk 1]
```diff
@@ -1231,8 +1231,8 @@ private int processHeadCreate(SCMHeadEvent<?> event, TaskListener global, String
                         listener = new LogTaskListener(LOGGER, Level.FINE);
                     }
                     long start = System.currentTimeMillis();
-                    listener.getLogger().format("[%tc] Received %s event with timestamp %tc%n",
-                            start, eventType, eventTimestamp);
+                    listener.getLogger().format("[%tc] Received %s %s event with timestamp %tc%n",
+                            start, eventClass, eventType, eventTimestamp);
```
Comment (jglick) [inline src/main/java/jenkins/branch/MultiBranchProject.java]: Would help reviewers if the PR summary were to give an example of the expanded log message, relative to the original.

Comment (stephenc) [inline src/main/java/jenkins/branch/MultiBranchProject.java]:
```
[Tue Jan 24 16:32:24 UTC 2017] Received UPDATED event with timestamp Tue Jan 24 16:32:24 UTC 2017
```

becomes

```
[Tue Jan 24 16:32:24 UTC 2017] Received org.jenkinsci.plugins.github_branch_source.PullRequestGHEventSubscriber$SCMHeadEventImpl UPDATED event with timestamp Tue Jan 24 16:32:24 UTC 2017
```

**Comment by aheritier** [comment]
🐝

**Comment by HRMPW** [comment]
Does anyone have any objections or see risk in putting this in the release? @jglick

**Pull request merged**